tlm_teamd/main.cpp

```diff
@@ -30,6 +31,11 @@ void update_interfaces(swss::SubscriberStateTable & table, TeamdCtlMgr & mgr)
         const auto & lag_name = kfvKey(entry);
         const auto & op = kfvOp(entry);
 
+        if (lag_name.find(VLAN_SUB_INTERFACE_SEPARATOR) != std::string::npos)
```

Comment:
This looks fine to me. 
A question to see if there is any interface type property which can be used to identify that this is a subinterface ?

Reply:
In current design there is no interface type property. I do not see any other interface type under portchannel other than subinterface having "." in its name.

Reply:
Agree with @preetham-singh . There is no property currently to identify as a sub-interface